modules/renter/contractor/contractor.go

```diff
@@ -6,36 +6,22 @@ import (
 	"path/filepath"
 	"sync"
 
-	"github.com/NebulousLabs/Sia/crypto"
 	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/modules/renter/contractor/proto"
```

Comment:
too much nesting. Fine for now but we should figure out a better way to structure it. Probably file to have it all in /renter